starter_code/queries.md

```diff
@@ -51,33 +57,37 @@
 <!-- Your Code Goes Here -->
 
 ### 12. All the companies founded before 2000 that have an acquisition amount of more than 10.000.000
+filter: {"acquisition.price_amount":{$gte:10000000},founded_year:{$lt:2000}}
 
-<!-- Your Code Goes Here -->
 
 ### 13. All the companies that have been acquired after 2010, order by the acquisition amount, and retrieve only their `name` and `acquisition` field.
-
-<!-- Your Code Goes Here -->
+filter {"acquisition.acquired_year":{$lt:2010}}
+project {name:1,acquisition:1}
+sort {"acquisition.price_amount":1}
 
 ### 14. Order the companies by their `founded year`, retrieving only their `name` and `founded year`.
 
-<!-- Your Code Goes Here -->
+project: {name:1,founded_year:1}
+sort: {founded_year:1}
 
 ### 15. All the companies that have been founded on the first seven days of the month, including the seventh. Sort them by their `acquisition price` in a descending order. Limit the search to 10 documents.
 
-<!-- Your Code Goes Here -->
+filter: {$and: [{founded_day:{$lte:7}}, {founded_month: 1}]}
+sort : {"acquisition.price_amount":-1}
 
 ### 16. All the companies on the 'web' `category` that have more than 4000 employees. Sort them by the amount of employees in ascending order.
 
-<!-- Your Code Goes Here -->
 
+filter: {category_code:"web",number_of_employees:{$gt:4000}}  
+sort: {number_of_employees:1}
 ### 17. All the companies whose acquisition amount is more than 10.000.000, and currency is 'EUR'.
-
-<!-- Your Code Goes Here -->
+    filter: {"acquisition.price_amount":{$gt:10000000},"acquisitions.price_currency_code":"EUR" }
 
 ### 18. All the companies that have been acquired on the first trimester of the year. Limit the search to 10 companies, and retrieve only their `name` and `acquisition` fields.
-
-<!-- Your Code Goes Here -->
+filter :{founded_month:{$lte:3}}
+project: {name:1,acquisition:1}
+Limit : 10
 
 ### 19. All the companies that have been founded between 2000 and 2010, but have not been acquired before 2011.
 
-<!-- Your Code Goes Here -->
+Filter: {founded_year: {$gte:2000, $lte:2010},"acquisitions.acquired_year":{$lt:2011}}
```

Comment:
don't forget the and:
{$and: [{founded_year: {$gt:2000, $lt:2010},"acquisitions.acquired_year":{$lt:2011}}]}